doc/terminology.md

```diff
@@ -0,0 +1,106 @@
+# Terminology
+
+This pages gives an overview about the terms we use, when talking about interacting with VSS signals in-vehicle using KUKSA.val components.
+
+* [VSS in-vehicle: Main components](#vss-in-vehicle-main-components)
+   * [VSS Ressource Server](#vss-ressource-server)
+   * [Client](#client)
+   * [Clients: VSS Consumers](#clients-vss-consumers)
+   * [Clients: VSS Providers](#clients-vss-providers)
+      * [data-provider](#data-provider)
+      * [actuation-provider](#actuation-provider)
+* [Vehicle Signal Specification (VSS)](#vehicle-signal-specification-vss)
+   * [Signal](#signal)
+      * [Sensor](#sensor)
+      * [Actuator](#actuator)
+      * [Attribute](#attribute)
+   * [Value](#value)
+   * [Metadata](#metadata)
+   * [Overlay](#overlay)
+   * [Datapoint](#datapoint)
+# VSS in-vehicle: Main components
+
+![In-vehicle VSS components](./pictures/terminology.svg)
+
+KUKSA.val helps you providing and interacting with VSS data in-vehicle. The figure above depicts the required components. We will describe each of their roles in the following sections. 
+## VSS Ressource Server
```

Comment:
```suggestion
# KUKSA.VAL Components

![In-vehicle VSS components](./pictures/terminology.svg)

KUKSA.val helps you provide and interact with VSS data in-vehicle. The figure above depicts the required components. We will describe each of their roles in the following sections. 
## Server
```

Just as we specify _Client_ below and go on to describe the different types etc, I think _Server_ is a better fit here (i.e. as top level heading). It could be good to specify what a "resource server" is also, especially pointing out the definition of it in OAuth2. I could also make sense to add another section for the authorization related terms, just like the _Vehicle Signal Specification_ section.

Reply:
I suggest we might decide to add the OAuth definitions in a later PR, if it seems to make sense.